pom.xml

```diff
@@ -25,6 +31,16 @@
 
 	<developers>
 		<developer>
+			<id>emsa23</id>
+			<name>Frank Ittermann</name>
+			<email>frank.ittermann@data4life.care</email>
+			<roles>
+				<role>developer</role>
+				<role>maintainer</role>
+			</roles>
+			<timezone>CET</timezone>
+		</developer>
+		<!--  Removed not to show on Wiki <developer>
```

Comment:
FTR https://issues.jenkins-ci.org/browse/WEBSITE-658 should solve it by removing dependency on Maven metadata.